scapy-ssl_tls/ssl_tls.py

```diff
@@ -8,7 +8,7 @@
 import os, time
 
 try:
-    import ssl_tls_crypto
+    from scapy_ssl_tls import ssl_tls_crypto
```

Comment:
Hi Eric,

Yes, this seems to finally solve the problem.
I tested the code after this change, and the results seem unaffected. I
pushed this change, fingers crossed.

thanks a lot,
Nimrod


On Wed, Mar 2, 2016 at 1:40 AM, Eric Lordahl <notifications@github.com>
wrote:

> In scapy-ssl_tls/ssl_tls.py
> <https://github.com/nimia/public_drown_scanner/pull/7#discussion_r54655838>
> :
>
> > @@ -8,7 +8,7 @@
> >  import os, time
> >
> >  try:
> > -    import ssl_tls_crypto
> > +    from scapy_ssl_tls import ssl_tls_crypto
>
> This line resoles the following errors for me:
>
> Import Error - most likely due to missing pycrypto libraries - disabling
> crypto functionality
> ImportError('cannot import name TLSHandshake',). Perhaps it can be split
> from this PR?
>
> [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nimia/public_drown_scanner/pull/7/files#r54655838>.
>